protostar/starknet/forkable_starknet.py

```diff
@@ -31,6 +31,14 @@ async def empty(
             state=await CheatableStarknetState.empty(general_config=general_config)
         )
 
+    @classmethod
+    async def from_contract_class(
```

Comment:
Why do we need this?

Reply:
It can be removed though because it doesn't do much. I tried various ways of reducing the boilerplate caused by creating a StarkNet instance.